cmd/agent/app/check.go

```diff
@@ -131,6 +131,10 @@ var checkCmd = &cobra.Command{
 			fmt.Println(string(checkStatus))
 		}
 
+		if checkRate == false {
+			color.Yellow("Currently only displaying Gauge metrics. Please run the command with --check-rate to see any other metric types if available.")
```

Comment:
Can we s/Please/You can ?
"Please" sounds to me like the user did something wrong that need to fix, does it make sense?

Reply:
Good call, definitely don't want to imply something was done wrong. Updated! 🙇